assets/apps/customizer-controls/src/builder/scss/_typography-extra-section.scss

```diff
@@ -0,0 +1,19 @@
+.control-section-typography_extra_section {
+  display: list-item !important;
+
+  .neve-white-background-control {
+    background: none !important;
+    padding: 15px 30px;
+    position: relative;
+  }
+
+  label {
+    color: rgb(80, 87, 94);
+  }
+
+  .customize-control-description {
+    position: absolute;
+    right: 30px;
+    top: 6px;
+  }
+}
```

Comment:
Hey @abaicus can you look here closer against is this place ok or is it like a hack? Thank you.

Reply:
@mskapusuz a good solution would be to extend the section constructor and overwrite the `isContextuallyActive` & `attachEvents` methods. This way, the section won't be affected by any unexpected events or callbacks the customizer might fire.

```js
wp.customize.sectionConstructor['typography_extra_section'] = wp.customize.Section.extend({
	// No events for this type of section.
	attachEvents: () => null,
	// Always make the section active.
	isContextuallyActive: () => true,
});
```

Reply:
Great. Thank you @abaicus for your review!